lib/pbench/test/unit/server/conftest.py

```diff
@@ -17,7 +17,7 @@
 from freezegun import freeze_time
 import jwt
 import pytest
-from requests import Response
+from werkzeug.wrappers.response import Response
```

Comment:
We usually use `from flask.wrappers import Response` and consistency would be good, but functionally it's the same thing.